mmaction/utils/gradcam_utils.py

```diff
@@ -1,5 +1,6 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 import torch
+import torch.nn as nn
```

Comment:
What about moving this to 'visualization/grad_cam.py'?

Reply:
I will reorganize visualzation tools structure in pr #2092